docs/customization/README.md

```diff
@@ -159,6 +159,47 @@ module.exports = {
 };
 ```
 
+## Mutating `neutrino.options`
+
+While it is possible to mutate `neutrino.options` directly, this should be avoided.
+Instead, it is _always recommended_ to pass an `options` object to ensure proper normalization.
+
+```js
+// Bad: Using function format, overriding `neutrino.options` properites.
+// Paths will not be relative to `neutrino.options.root` as expected.
+module.exports = neutrino => {
+  Object.assign(neutrino.options, {
+    source: 'lib'
+  });
+
+  neutrino.options.output = 'dist';
```

Comment:
This line here is actually a supported way of changing some of the options. Each of these is a getter/setter, so setting the value to a relative path let's you use a getter for the absolute path. Same for `mains`, you can get/set them just fine.

The problem is using `Object.assign` or merge on them, as this *overwrites* the property definition.

I'd change this to highlight this technique as "good".

Reply:
Ah, wondered about that - I'll change it.